src/components/TreeDetails.tsx

```diff
@@ -137,6 +137,35 @@ const TreeDetails: React.FC<TreeDetailsProps> = ({ treeID }) => {
   return (
     <>
       {/* <IonListHeader>Overview</IonListHeader> */}
+      <IonCard>
+        <IonCardHeader>
+          {/* <IonCardTitle class="ion-text-center">LIDAR Scans</IonCardTitle> */}
+        </IonCardHeader>
+        <IonCardContent>
+          {/* <IonItem lines="none">
+            <IonSegment value={"front"}>
+              <IonSegmentButton value="front">
+                <IonLabel>Front</IonLabel>
+              </IonSegmentButton>
+              <IonSegmentButton value="back">
+                <IonLabel>Back</IonLabel>
+              </IonSegmentButton>
+            </IonSegment>
+          </IonItem> */}
```

Comment:
Let's get always rid of these commented blocks as they are really confusing in some files and things like this are easy to reproduce

Reply:
I want to use this one, as soon as we have the separated images from Julian.